docs/configuration.md

```diff
@@ -42,7 +48,7 @@ clientConnection:
 
 # Whether or not to enable IPsec encryption of tunnel traffic. IPsec encryption is only supported
 # for the GRE tunnel type.
-enableIPSecTunnel: true
+#enableIPSecTunnel: true
```

Comment:
Not related to this PR, the default value of enableIPSecTunnel should be false, this was set by mistake, could you also correct it with this PR?

Reply:
Sure I will